lib/github_api/fetcher.rb

```diff
@@ -37,7 +38,11 @@ def check_latest_commit
 
       unless results.errors.any?
         commits = results.data.repository.ref.target.history.edges
-        assign_badge if commits.length == 1 && commits.first.node.authoredByCommitter
+
+        if commits.length == 1
+          assign_badge if commits.first.node.authoredByCommitter
+          @project.update_column :last_commit, commits.first.node.committedDate
```

Comment:
@DMips here the value the `last_commit` has is actually the commit date for the first commit. Does it need to be the latest commit instead?